ProjectSettings/ProjectVersion.txt

```diff
@@ -1,2 +1,2 @@
-m_EditorVersion: 2019.4.15f1
-m_EditorVersionWithRevision: 2019.4.15f1 (fbf367ac14e9)
+m_EditorVersion: 2019.4.17f1
+m_EditorVersionWithRevision: 2019.4.17f1 (667c8606c536)
```

Comment:
Nice!
Could you also update the link in the docs? b093f572fc6fb3fa08f69e1f8c28d49c4aacd26f

Reply:
Done... I hope :)

Reply:
Seems you opened a new PR for that #2026. I can stitch them together if you want 🙂

Reply:
Yes please, I missclicked :)